src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -283,10 +283,10 @@ grpc_error* Chttp2ServerListener::Create(Server* server, const char* addr,
     grpc_error* error = GRPC_ERROR_NONE;
     if (absl::StartsWith(addr, "unix:")) {
       error =
-          grpc_resolve_unix_domain_address(addr + sizeof("unix:"), &resolved);
+          grpc_resolve_unix_domain_address(addr + strlen("unix:"), &resolved);
```

Comment:
I think we could still use `sizeof()` here if we defined the string as a constant, like this:

```
const char kUnixUriPrefix[] = "unix:";
```

Then we could use `kUnixUriPrefix` on line 284 above, and then `sizeof(kUnixUriPrefix)` here.

Same thing for the `unix-abstract:` prefix below.

Reply:
```"unix:" = {'u', 'n', 'i', 'x', ':', '\n'}
sizeof("unix:") = 6
strlen("unix:") = 5```

I made it a constant. Let me know if you would prefer
`const char kUnixUriPrefix[] = {'u', 'n', 'i', 'x', ':'}` and `sizeof`

Reply:
Then let's use `sizeof(kUnixUriPrefix) - 1`.  The reason why `sizeof()` is better than `strlen()` is that `sizeof()` is evaluated at compile-time, whereas `strlen()` has to iterate through the string to find the NUL at run-time.

Reply:
Done.